### PR TITLE
[LXR] Fix checkout-version.py for python3

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
hold
Running job now, let's merge once it's successful

File: lxr/checkout-version.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (smuzaffar) [inline lxr/checkout-version.py:1]: @iarspider , do we have python3 on current cmslxr node?

Comment (iarspider) [reply lxr/checkout-version.py:1]: Yes

**Comment by smuzaffar** [comment]
+externals
https://cmssdt.cern.ch/jenkins/job/lxr-generate-index/4195/consoleFull finished successfully

**Pull request merged**